adoc/admin-monitoring-stack.adoc

```diff
@@ -667,6 +676,14 @@ helm install --name grafana suse/grafana \
 --namespace monitoring \
 --values grafana-config-values.yaml
 ----
+Or if you have selected the Helm 3 alternative <<helm-tiller-install>>:
++
+[source,bash]
+----
+helm install grafana suse/grafana \
+--namespace monitoring \
```

Comment:
If the namespace does not exist, it will create it in helm2. But in helm3, users need to create namespace before install charts in the namespace.

Reply:
Aha, fortunately the admin-monitoring-stack.adoc already has instructions for that on line 91. :)  I'll check the other documents as well.

Reply:
I found a couple other spots where this could be a problem, including Stratos instructions. I have modifed them and they are in the PR now.